src/model.rs

```diff
@@ -201,6 +202,21 @@ pub struct FSRS<B: Backend = NdArrayBackend> {
     device: B::Device,
 }
 
+#[wasm_bindgen]
+pub struct FSRSwasm(FSRS<NdArrayBackend>);
```

Comment:
`wasm_bindgen` [can't handle generics](https://github.com/rustwasm/wasm-bindgen/issues/3309), so I made it concrete.